test/IDE/complete_multiple_trailingclosure.swift

```diff
@@ -184,3 +185,18 @@ func testOptionalInit() {
 // INIT_REQUIRED_NEWLINE_3-NOT: name=fn3
 // INIT_REQUIRED_NEWLINE_3: End completions
 }
+
+struct MyStruct4<T> {
+  init(arg1: Int = 0, arg2: () -> T) {}
+
+  func testStructMethod() {}
+}
+func testFallbackPostfix() {
+  MyStruct4 {
+    1
+  } #^INIT_FALLBACK^#
```

Comment:
Is it possible to test without a trailing closure, or is that the only time it reproduces?  Also, maybe a test that uses a non-default argument as well.

Reply:
The problem only happens after trailing closures because otherwise they are parsed as normal postfix expression completions.

Added a test case with non-defaulted arguments